gems/sorbet-runtime/lib/types/types/fixed_array.rb

```diff
@@ -6,23 +6,29 @@ module T::Types
   # Takes a list of types. Validates each item in an array using the type in the same position
   # in the list.
   class FixedArray < Base
-    attr_reader :types
-
     def initialize(types)
-      @types = types.map {|type| T::Utils.coerce(type)}
+      @inner_types = types
+    end
+
+    def types
+      @types ||= @inner_types.map {|type| T::Utils.coerce(type)}
+    end
+
+    def build_type
```

Comment:
What do you think about explicitly returning `nil` in these methods so we don't accidentally expose a contract that we can't change in the future?

Reply:
Good point, done 👍